.openapi-generator/FILES

```diff
@@ -141,7 +141,6 @@ docs/UpdateConferenceRequest.md
 docs/UpdateConferenceRequestStatus.md
 freeclimb.sln
 git_push.sh
-src/freeclimb.Test/Api/DefaultApiTests.cs
```

Comment:
why is this removed?

Reply:
This wasn't auto-generated so it got removed, this keeps a list of all files auto-generated, since there are no changes with the default api tests file, it got removed.